examples/advanced_circuits_algorithms/Deutsch_Jozsa_Algorithm/Deutsch Jozsa Algorithm.ipynb

```diff
@@ -0,0 +1,1071 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.        single_gate_set = "x"
</pre><p>It's easier to work with if the list contains the gate itself:</p><pre class="ql-syntax" spellcheck="false">single_gate_set = [gates.X()]
</pre><p>This way, you can programmatically attach gates to the circuit:</p><pre class="ql-syntax" spellcheck="false">circuit.add_instruction(Instruction(random.choice(single_gate_set), qubit))
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/177/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>